src/Orchard.Web/Modules/Orchard.Users/Views/Account/ChangePasswordSuccess.cshtml

```diff
@@ -1,3 +1,6 @@
 ﻿@model dynamic
 <h1>@Html.TitleForPage(T("Change Password").ToString())</h1>
 <p>@T("Your password has been changed successfully.")</p>
+@if ((bool)ViewData["InvalidateOnPasswordChange"]) {
```

Comment:
Unnecessary. Users are redirected to the login page, it's obvious they need to use the new one.

Reply:
While trying the feature, I was not being redirected anywhere, so I added this paragraph. Could be the redirect is not in place anymore?

Reply:
Good then.